convenience

```diff
@@ -17,6 +17,9 @@ case "$OSTYPE" in
     cygwin)
         p="WINDOWS"
         ;;
+    msys)
+        p="WINDOWS"
+        ;;
```

Comment:
Good idea on the `cygwin|msys` simplification.  I was just trying this under my Git Bash shell that ships with git, and this was all that was reqiured to get it to work

I don't think we should detect WSL,  WSL is linux and we should treat it as such.  I really don't like the idea of automatically attempting to defeat linux detection using conditionals like this. I fear it will evolve into the maintainability thicket that is cygwin with special cases scattered throughout the codebase.  One of the primary ways I hope to use WSL is to produce linux builds of the viewer and 3p packages.  We should probably provide support and testing around specifying something like autobuild's `-p windows` when running under WSL, but I think that should be based on explicit user input to configure a windows build intentionally, in which case detection is moot.

Reply:
> I don't think we should detect WSL, WSL is linux and we should treat it as such. One of the primary ways I hope to use WSL is to produce linux builds of the viewer and 3p packages.

That's an interesting perspective, worth considering. I brought it up only because there have been questions before of the form, "Now that Windows has WSL that can run bash scripts on Windows, why must we still use cygwin?" -- which is the use case I had in mind. But you're correct that we must remain aware of both potential use cases.

Reply:
Yeah, I strongly suspect that WSL bash would be a worse "bash for windows" solution than cygwin.  I'm all for eliminating cygwin requirements, but I think Git's msys is probably better for that purpose.